components/SocialIcons/SocialIcons.tsx

```diff
@@ -0,0 +1,30 @@
+import { InstagramIcon, FacebookIcon, TwitterIcon, YoutubeIcon,TiktokIcon,LinkedinIcon,PinterestIcon } from "./SocialSvgComponents";
```

Comment:
I would argue that this is an unnecessary component. I think that representing each svg as a component is enough abstraction for now. I suppose we could find ourselves placing this exact icon list at other parts of the site in the future; however, I think it would be better to just wait until that point. 

My recommendation is to get rid of this component and instead create an ordered list in `Footer`.